### DISCUSSION TITLE
Feature idea - start logic analyzer trigger after ms/ count

### DISCUSSION THREAD

**Discussion by marfrit** [fhdm-dev/scoppy#43] [General]
Another feature idea I've had is that it would be cool if the trigger was enabled n milliseconds after the button was clicked or recoding only starts after n observations. 

Example - the setup of my keyboard takes about 2.5 seconds and it is a bit tricky to see the part I'm interested in, since it will post about 88 clock cycles of keyboard reset info back and forth until the part I'm interested in.

**Comment by fhdm-dev** [comment]
> Another feature idea I've had is that it would be cool if the trigger was enabled n milliseconds after the button was clicked or recoding only starts after n observations.
> 
> Example - the setup of my keyboard takes about 2.5 seconds and it is a bit tricky to see the part I'm interested in, since it will post about 88 clock cycles of keyboard reset info back and forth until the part I'm interested in.

Yep. Good idea. I'll have a look at what options real logic analyzers have in this regard and see what I can incorporate into Scoppy.
